docs/sql/array.md

```diff
@@ -36,11 +39,24 @@ as `CREATE TABLE data AS (CITIES VARCHAR ARRAY, COUNTRY VARCHAR)`.
+| `ARRAY_APPEND(` _array_ `,  ` _element_ `)`                          | Appends an element to the end of the array and returns the result. If the array is `NULL`, the function will return `NULL`. If the element is `NULL`, the `NULL` element will be added to the end of the array.                                                                                                                         | `ARRAY_APPEND(ARRAY [1, 2], 3)` => `[1, 2, 3]`                      |
+| `ARRAY_PREPEND(` _array_ `,  ` _element_ `)`                         | Prepends an element to the start of the array and returns the result. If the array is `NULL`, the function will return `NULL`. If the element is `NULL`, the `NULL` element will be added to the start of the array.                                                                                                                    | `ARRAY_PREPEND(ARRAY [2, 3], 1)` => `[1, 2, 3]`                     |
+| `ARRAY_COMPACT(` _array_ `)`                                         | Returns a new array by removing all the `NULL` values from the _array_                                                                                                                                                                                                                                                                  | `ARRAY_COMPACT(ARRAY [1, 2, 3, NULL])` => `[1, 2, 3]`               |
+| `SORT_ARRAY(` _array_ `, [` _ascendingOrder_ `])`                    | Returns a new array, sorted in ascending or descending order according to the natural ordering of the array elements. The default order is ascending if ascendingOrder is not specified. Null elements will be placed at the beginning of the returned array in ascending order or at the end of the returned array in descending order | `SORT_ARRAY(ARRAY [4, 7, 1, null])` => `[null, 1, 4, 7]`            | 
+| `ARRAY_MAX(` _array_ `)`                                             | Returns the maximum value in the _array_                                                                                                                                                                                                                                                                                                | `ARRAY_MAX(ARRAY [9, 1, 2, 4, 8, null])` => 9                       | 
+| `ARRAY_MIN(` _array_ `)`                                             | Returns the minimum non null value in the _array_                                                                                                                                                                                                                                                                                       | `ARRAY_MIN(ARRAY [9, 1, 2, 5, 8, null])` => 1                       |
```

Comment:
you have to say that this ignores null, otherwise based on the definition of sort we could guess that the result is null